presto-main/src/main/java/io/prestosql/operator/PipelineStats.java

```diff
@@ -344,18 +344,33 @@ public boolean isFullyBlocked()
         return blockedReasons;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the physical input data size in bytes.
```

Comment:
It's not clear to me what is this "physical input data". Is any other way to make it clear for the reader?